source/unified-test-format/unified-test-format.rst

```diff
@@ -475,11 +475,12 @@ The structure of this object is as follows:
     connecting to multiple mongos servers is necessary to test session
     pinning.
 
-    If this option is true or unset and the topology type is
-    ``LoadBalanced``, the test runner MUST connect the MongoClient to a load
-    balancer that is balancing multiple servers. If it is false and the
-    topology type is ``LoadBalanced``, the test runner MUST connect the
-    MongoClient to a load balancer that is only fronting a single server.
+    If the topology type is ``LoadBalanced``, the test runner MUST use one of
```

Comment:
I just noticed that the spec refers to "load-balanced" (`runOnRequirements`), "LoadBalanced" (prose sections), and "loadBalanced" (changelog). Can we standardized on "load-balanced" everywhere to agree with the topology type referenced in `runOnRequirements`?

Reply:
The actual topology type in the SDAM spec will be called `LoadBalanced` so I'd like to keep the prose as-is. For `runOnRequirements`, I used `load-balanced` because the existing topologies requirements are all lowercase and `sharded-replicaset` sets a precedent for hyphenating multiple words. The changelog entry was referring to an older change that has since been removed so I updated that to remove the `loadBalanced` wording.

Reply:
Works for me. Thanks for the explanation.